src/cockpit/packages.py

```diff
@@ -168,10 +171,11 @@ def __init__(self, path):
         # HACK: drop this after getting rid of ${libexecdir}, see above
         self.manifest = patch_libexecdir(self.manifest)
 
-        self.name = self.manifest.get('name', path.name)
-        self.content_security_policy = None
-        self.priority = self.manifest.get('priority', 1)
-        self.bridges = self.manifest.get('bridges', [])
+        with safely.catch(str(manifest_path), logger.warning):
+            self.content_security_policy = safely.get(self.manifest, 'content-security-policy', str, '')
+            self.name = safely.get(self.manifest, 'name', str, path.name)
+            self.priority = safely.get(self.manifest, 'priority', int, 1)
+            self.bridges = safely.get_list(self.manifest, 'bridges', dict, [])
```

Comment:
safely.catch() does not really catch an exception, it just logs and re-throws it. So, shouldn't these still be handled here, and a broken manifest gets ignored instead of crashing the bridge?